config/locales/fil/about.fil.yml

```diff
@@ -0,0 +1,22 @@
+    p3header: 'Bakit mo pinili ang pangalang REFUGE?'
+    p3: 'Matatag ang aming paniniwala na ang lahat ay mayroong karapatan na gumamit ng pahingaan nang ligtas at gusto namin na ang pangalan ng aming aplikasyon ay magkaroon ng dignidad na kapareha sa binibigay namin sa aming mga taga-gamit. Sa madaling salita, kami ay umaasa na makapagbigay ng isang lugar na kanlungan para sa oras ng inyong pangangailangan.'
+    p4header: "Ano ba ang malaking importansiya tungkol sa mga banyo/pahingaan sa kahit ano at bakit natin kailangan ang ganitong kaalaman?"
+    p4: "Isa sa pinakamalaking isyu tungkol sa paglaban para sa karapatan ng mga transgender ay ang pagkakaroon ng sariling mga pahingaan o banyo. Tila ba na bawat linggo ang transgender ay nasa sentro ng pambansang kwento ng balita dahil sa paggamit ng pahingaan o banyo na nakatalaga sa tamang pagkakakilanlan o kasarian. Kaya naman, tayo ay naniniwala na bawat transgender na tao ay mayroong karapatan na gumamit ng pahingaan o banyo kung gustuhin man nila. Gayunpaman, ating napagtanto na sa kabila ng tagumpay sa batas ngayong taon ng tungkol sa paggamit ng pahingaan o banyo, ay marami paring indibidwal na mga transgender ang nakararanas ng berbal at pisikal na pang-aabuso nangdahil lamang sa pahingaan o banyo. Walang nakakapagpahayag nito - at dahil diyan ginawa namin itong REFUGE."
+    p5header: 'Anu ang pwede nating gawin para makatulong?'
+    p5:
+      first: "<strong>Una:</strong> Magdagdag ng mga listahan. Ang database ay maging kasing laki kapag ginawa mo ito. Kapag maraming mga nasa listahan, ay mas magiging komprehensibo at mas magkakaroon ng kahalagahan ang pamamaraang ito."
+      second: "<strong>Ikalawa:</strong> Ikalat ang mga salita. Tweet. Facebook. Blog. Kahit anong gustuhin mo, gawin mo. Ipaalam mo sa lahat ng tao ang tungkol sa pamamaraang ito."
+      third: "<strong>Ikatlo:</strong> Kung alam mo kun paano mag-code, bumisita sa GitHub at ipaalam sa amin ang tungkol sa bug, magmungkahi ng mga pagpapahusay, o kaya mag-ambag ng kunting code at tumulong sa proyektong ito. Ang REFUGE ay isang open source at hindi namin ito magagawa kung wala kayo."
+      fourth: "<strong>Ikaapat: </strong> Mag-ambag. Maging mapagmatyag sa darating na pangangalap ng pondo mula sa mga tao para makapaglikha ng pondo sa iilang teknolohiya na kinakailangan gamitin pati narin ang pagbayad sa mga taga-desinyo at mga inhinyero kahit sa maliit na halaga para sa kanilang sipag at walang pagud na paggawa. Sila ay nagtratrabaho ng walang bayad para makapaghatid ng serbisyo sa inyo at hindi namin gusto na magtratrabaho ang sinuman ng libre. Karamihan sa aming grupo ay transgender at mga walang trabaho sa ngayon."
+    p6header: "Bakit nakalista ang aking negosyo o organisasyon?"
+    p6: "Alamin ang higit pa sa aming <a href='/business_info'>Business FAQ</a>!"
```

Comment:
Related: Is there an appropriate translation for "business FAQ"?

(It would be okay to use the same translation as for "business info," if that works better than literally translating "FAQ")

I understand that some words are popular as "loan words" from other languages. If you feel this is best left as it already is, that is fine. My first goal is to have the most people understand. (And as a second guiding principle, to preserve the original tone/meaning.)